src/main/java/org/apache/maven/resolver/internal/ant/Names.java

```diff
@@ -19,22 +19,38 @@
 package org.apache.maven.resolver.internal.ant;
 
 /**
+ * Centralized constants used throughout the Maven Resolver Ant tasks.
+ * <p>
+ * This class defines symbolic names for commonly used IDs, property keys,
+ * and filenames used in the integration with Ant, such as default repository identifiers,
+ * offline mode flags, and POM registration keys.
+ * </p>
+ *
+ * <p>
+ * This class is not intended to be instantiated.
+ * </p>
  */
 public final class Names {
 
     private Names() {
         // hide constructor
     }
 
+    /** ID namespace prefix used by the resolver: {@code "resolver"}. */
```

Comment:
I don't know what an "ID namespace prefix" is. Is it just a namespace prefix?

Reply:
Yes, its just a namespace prefix. It is used in AntRepoSys.getInstance(..). The constant name is ID. Does removing ID make it clearer?

Reply:
Yes, it should just be "namespace prefix" then. "ID namespace prefix" is not common terminology

Reply:
OK fixed.